cmake/modules/extensions.cmake

```diff
@@ -1204,6 +1205,7 @@ function(zephyr_linker_sources location)
   set(rodata_path        "${snippet_base}/snippets-rodata.ld")
   set(ramfunc_path       "${snippet_base}/snippets-ramfunc-section.ld")
   set(nocache_path       "${snippet_base}/snippets-nocache-section.ld")
+  set(pm_sections_path   "${snippet_base}/snippets-pm-sections.ld")
```

Comment:
you define a filename `snippets-pm-sections.ld`, but I don't find this used anywhere by the build system or existing linker script files.

Where is this being used and how has it been tested ?

Reply:
We have an internal use for this that is going to be upstream soon. But as it is useful for targets doing complex power management it is worth to make it generic. The idea is that we can place symbols here that are located in memory that don't lose context during low power states.

Reply:
hopefully the internal solution you have will shed some light on the difference to SECTIONS, once you upstream the next part.

Reply:
Hey @tejlmand that is a great feedback, `SECTIONS` is good enough for the use cases I have in mind, if they converge for a common place in future I may review it. But for now using `SECTIONS` is good enough.

I'll close it since it is not need right now. Thanks a lot.